ext/xmlwriter/tests/003.phpt

```diff
@@ -5,7 +5,7 @@ XMLWriter: libxml2 XML Writer, membuffer, flush, attribute
 --FILE--
 <?php
 
-$doc_dest = '001.xml';
+$doc_dest = '003.xml';
```

Comment:
Missing unlink($doc_dest); ?

Reply:
these tests already exist, is it worth changing them?